src/Nova/EscaperoomSlot.php

```diff
@@ -79,10 +81,18 @@ public function fields(Request $request)
                 return $this->resource->formatted_start;
             })->exceptOnForms(),
             DateTime::make('Start At')->required(),
-            nova('rate') ? BelongsTo::make('Rate', 'rate', nova('rate'))->hideWhenCreating()->nullable() : null,
+            MorphTo::make('schedule')->types(array_filter([
+                nova('user'),
+                nova('contact'),
+                nova('customer'),
+                nova('order'),
+                nova('booking'),
+                nova('game'),
+                nova('block'),
+                nova('slot'),
```

Comment:
This doesn't look like the proper list for this morph.   The comment was only showing proper technique, not the desired set of objects.

Reply:
@drewroberts could you give me the nova resources related to this `morph` relation?

Reply:
Is this one resolved?

Reply:
Hey @joshtorres is missing the list of objects related to that morph relation, different than that is ready to merge